### PR TITLE
POST request test, Returning received data on _get_media()

### PR DESCRIPTION
**Pull request by epsooraj**

### CODE REVIEW

File: tests/server/test_nlu_server.py [hunk 1]
```diff
@@ -50,6 +50,20 @@ def test_classifier_server_train_get():
...
+    data = {'hotel': ['book a hotel', 'need a nice place to stay', 'any motels near by'],
+    'weather': ['what is the weather like', 'is it hot outside'],
+    'place': ['which place is this', 'where are we', 'where are you going', 'which place is it'],
+    'name': ['What is your name', 'Who are you', 'What do i call you', 'what are you called']
+    }
+    server = NLUServer().serve(test=True)
+    clf = Classifier()
+    for y, v in data.items():
+       for x in v:
+           server.requests.post("/models/classifier/train", json={"data": {"inputs": x, "targets": y}})
```
Comment (farizrahman4u) [inline tests/server/test_nlu_server.py]: Can you try posting the data at once, without the loop? i.e, `server.requests.post("/models/classifier/train", json={"data":data})`?

File: tests/server/test_nlu_server.py [hunk 2]
```diff
@@ -65,6 +79,20 @@ def test_entity_extractor_server_train_get():
...
     assert r.json() == ex.predict(test_inp)
 
+def test_entity_extractor_server_train_post():
+    X = ['what is the weather in tokyo', 'what is the weather', 'what is the weather like in kochi']
+    Y = [{'intent': 'weather', 'place': 'tokyo'}, {'intent': 'weather', 'place': 'here'}, {'intent': 'weather', 'place': 'kochi'}]
+    server = NLUServer().serve(test=True)
+    ex = EntityExtractor()
+    ex.fit(X, Y)
+    for x, y in zip(X, Y):
+        r = server.requests.get("/models/entity_extractor/train", json={"data":{"inputs":x, "targets":y}})
```
Comment (farizrahman4u) [inline tests/server/test_nlu_server.py]: Train with whole data at once, no loop.

**Comment by epsooraj** [comment]
Training with POST request without loop

**Comment by farizrahman4u** [comment]
there is still loopin clf test

**Comment by epsooraj** [comment]
Removing all Loops

**Comment by farizrahman4u** [comment]
LGTM, Thanks.

**Pull request merged**